frontend/src/components/landing/PlanMatrix.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from "@testing-library/react";
+
+import { mockLocalizedModule } from "../../../__mocks__/components/Localized";
+import { mockConfigModule } from "../../../__mocks__/configMock";
+import { setMockProfileData } from "../../../__mocks__/hooks/api/profile";
+import {
+  setMockRuntimeDataOnce,
+  getMockRuntimeDataWithPhones,
+  getMockRuntimeDataWithBundle,
+  getMockRuntimeDataWithoutPremium,
+} from "../../../__mocks__/hooks/api/runtimeData";
+import { mockUseFxaFlowTrackerModule } from "../../../__mocks__/hooks/fxaFlowTracker";
+import { mockUseL10nModule } from "../../../__mocks__/hooks/l10n";
+import { mockNextRouter } from "../../../__mocks__/modules/next__router";
+import { mockReactGa } from "../../../__mocks__/modules/react-ga";
+
+jest.mock("next/router", () => mockNextRouter);
+jest.mock("react-ga", () => mockReactGa);
+jest.mock("../../config.ts", () => mockConfigModule);
+jest.mock("../../hooks/gaViewPing.ts");
+jest.mock("../../hooks/gaEvent.ts");
+jest.mock("../../hooks/fxaFlowTracker.ts", () => mockUseFxaFlowTrackerModule);
+jest.mock("../../hooks/l10n.ts", () => mockUseL10nModule);
+jest.mock("../../components/Localized.tsx", () => mockLocalizedModule);
```

Comment:
reminder (non-blocking): we'll try to refactor these into shared frontend/__mocks__ when we work on frontend test coverage again.